content/php/concepts/math-functions/terms/round/round.md

```diff
@@ -0,0 +1,60 @@
+# Title:     _round function in php_
+
+## Description
+    The round() function rounds a floating-point number.
+    
+## Tips:
+    | To round a number UP to the nearest integer, look at the ceil() function.
+    | To round a number DOWN to the nearest integer, look at the floor() function.
+    | Return type of round function is float
+
+##  Syntax
+```pseudo
+round(number,precision,mode);
+```
+
+### Parameter:
+
+#### The `round()` function takes three parameter:
+
+| Parameter | Description |
+| ------ | ------ |
+| number | Required. Specifies the value to round |
+| precision | Optional. Specifies the number of decimal digits to round to. Default is 0 |
+| mode | Optional. Specifies a constant to specify the rounding mode like examples below.|
+
+
+#### Mode parameter
+
+- [ **PHP_ROUND_HALF_UP** ] - Default. Rounds number up to precision decimal, when it is half way there. </br> Rounds 1.5 to 2 and -1.5 to -2
+- [ **PHP_ROUND_HALF_DOWN** ] - Round number down to precision decimal places, when it is half way there. </br> Rounds 1.5 to 1 and -1.5 to -1
+- [ **PHP_ROUND_HALF_EVEN** ] - Round number to precision decimal places towards the next even value
+- [ **PHP_ROUND_HALF_ODD** ] - Round number to precision decimal places towards the next odd value
```

Comment:
[Proofreading] Check out my feedback on this [line](https://github.com/Codecademy/docs/pull/2531/files#r1234568067) to see how to improve this part of your entry. 
```suggestion
- [ **PHP_ROUND_HALF_UP** ] - Default. Rounds the number up to a precision decimal, when it is halfway there. </br> Rounds 1.5 to 2 and -1.5 to -2
- [ **PHP_ROUND_HALF_DOWN** ] - Rounds the number down to precision decimal places, when it is halfway there. </br> Rounds 1.5 to 1 and -1.5 to -1
- [ **PHP_ROUND_HALF_EVEN** ] - Rounds the number to precision decimal places towards the next even value
- [ **PHP_ROUND_HALF_ODD** ] - Rounds the number to precision decimal places towards the next odd value
```